.github/workflows/ci.yml

```diff
@@ -65,3 +65,22 @@ jobs:
       run: make probe-locally
     - name: Bring down local DSS instance
       run: make down-locally
+    - name: Checkout monitoring repository
```

Comment:
We should be using uss_qualifier as a standalone tool that is released separately from the DSS.  When this is the case, we should simply be able to use the released `monitoring` docker image (like [the prober check](https://github.com/interuss/dss/blob/73c267551e3a047acb1fd53fc90c7d97efeba814/build/dev/probe_locally.sh#L43) does) and not check out the repo at all.  If we really need to use a specific commit of that repo that is not released, that probably indicates a problem in our release strategy since if this user needs that specific unreleased commit, it's likely others will want to use that specific unreleased commit as well, so we should make a `monitoring` release that includes that commit.